src/utils/mtev_watchdog.h

```diff
@@ -176,5 +193,14 @@ API_EXPORT(void)
 
 API_EXPORT(int)
   mtev_setup_crash_signals(void (*)(int, siginfo_t *, void *));
+
+/*! \fn void mtev_watchdog_disable_asynch_core_dump(void)
+    \brief Disable asynchronous core dumps.
+
+    Disable starting a new child while a faulting prior child is still dumping.  This will be
+    overridden by the MTEV_ASYNCH_CORE_DUMP is present.
```

Comment:
Is the "overridden" still true if I call this very late in the program execution?